<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointTable</class>
 <widget class="QWidget" name="PointTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1~8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_1">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>9~16</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLineEdit" name="lineEdit_14">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLineEdit" name="lineEdit_15">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLineEdit" name="lineEdit_16">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>17~24</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_17">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_18">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="lineEdit_19">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="lineEdit_20">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLineEdit" name="lineEdit_21">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QLineEdit" name="lineEdit_22">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QLineEdit" name="lineEdit_23">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QLineEdit" name="lineEdit_24">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_12">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>25~32</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_25">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEdit_26">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="lineEdit_27">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="lineEdit_28">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLineEdit" name="lineEdit_29">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QLineEdit" name="lineEdit_30">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QLineEdit" name="lineEdit_31">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QLineEdit" name="lineEdit_32">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
